Improve CONTRIBUTING.md file

adebisi4145 opened the pull request:
Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

**Description**
Improved CONTRIBUTING.md file by adding `table of contents`, `How to Contribute` heading  and `issue label` table

**Changes to be made**

- Add `table of contents` for easy navigation
- Create an `issue label` table
- Add `How to Contribute` heading
- Specify the different types of issues `🦠Model Request`, `🐛 Bug Report` and `📑 Feature Request`
- Add Emojis to each sub sections for better visual representation

**Status**

- I have successfully added `table of contents`

> ![TOC](https://github.com/user-attachments/assets/d7f97e91-8c51-440e-a993-cf625666e640)

- Added `issue label` table and added `How to Contribute` heading

> ![IssueLable](https://github.com/user-attachments/assets/bd9bdcba-33fb-45da-9c12-bd0d70666fd1)
- I added the issue types  `🦠Model Request`, `🐛 Bug Report` and `📑 Feature Request`
- I added handshake emoji 🤝 to `Code of Conduct` sub section since it signifies "cooperation and respect"
- I added compass emoji 🧭 to `Where to Start?` sub section since it signifies "guidance and direction"
- I added scroll emoji 📜 to `How to Contribute` sub section since it signifies "documentation and guidance"
- I added light bulb emoji 💡 to `Other Contributing Suggestions` sub section since it signifies "ideas and suggestions"
- I added question mark emoji ❓ to `I still have questions` sub section since it signifies "question or uncertainty"

New [CONTRIBUTING.md](https://github.com/user-attachments/files/17311179/CONTRIBUTING.md) file

Is this pull request related to any open issue? If yes, replace issueID below with the issue ID

Related to #1271

adebisi4145 commented:
Hello @DhanshreeA , I've made some improvevement to CONTRIBUTING.md file and the PR is ready for review

Please let me know if there are other areas i can further improve the file

DhanshreeA commented:
@adebisi4145 couple of ideas/comments: 

1. Once we merge the DEVELOPMENT.md file, we can include a link to that here as well. 
2. In the meantime, in the "Creat a new issue" section, you could also expand upon the Issue types we have configured here, especially, the `Model Request`, `Bug Report`, and `Feature Request` issues.
3. We could have emojis for each of the sub sections like the "Create a new issue", what do you think?

adebisi4145 commented:
> @adebisi4145 couple of ideas/comments:
> 
> Once we merge the DEVELOPMENT.md file, we can include a link to that here as well.
> In the meantime, in the "Creat a new issue" section, you could also expand upon the Issue types we have configured here, especially, the Model Request, Bug Report, and Feature Request issues.
> We could have emojis for each of the sub sections like the "Create a new issue", what do you think?


The use of emojis is a great idea @DhanshreeA, I will implement the changes, push and wait for DEVELOPMENT.md file to be merged 

Thanks for suggesting other areas I can further improve the file

adebisi4145 commented:
Hello @DhanshreeA , I have implemented the changes you suggested. I have added the issue types and the emojis

- For the `Code of Conduct` sub section, I used handshake emoji 🤝 emoji since it represent "cooperation and respect"
- For the `Where to Start?` sub section, I used compass emoji 🧭 since it represent "guidance and direction"
- For the `How to Contribute` sub section, I used scroll emoji 📜 since it represent "documentation and guidance"
- For the `Other Contributing Suggestions` sub section, I used light bulb emoji 💡 since it represent "ideas and suggestions"
- For the `I still have questions` sub section, I used question mark emoji ❓ since it signifies "question or uncertainty"

Malikbadmus commented on CONTRIBUTING.md:
```diff
@@ -4,27 +4,46 @@ Thank you for contributing to Ersilia's mission and vision! ⭐
+You can create the following issue types:
+* **🦠 Model Request:** Request to add a new model to the hub
+* **🐛 Bug Report:** Report something that is not working as expected
+* **📑 Feature Request:** Suggest a new feature or enhancement
```
@adebisi4145 , you only listed 3 issue types?

adebisi4145 replied:
Thank you for the feedback @Malikbadmus . Initially, I focused on the 3 @DhanshreeA mentioned as the most important, but I will go ahead and add the others to make it more comprehensive.

adebisi4145 replied:
Hello @Malikbadmus , I have implemented the changes

DhanshreeA commented:
Thank you @adebisi4145! 🥳

adebisi4145 commented:
> Thank you @adebisi4145! 🥳

Thank you for an opportunity to learn @DhanshreeA
